postgres/pixl-db_init.sh

```diff
@@ -25,4 +25,5 @@ psql -U "${POSTGRES_USER}" --dbname "${POSTGRES_DB}" -c "$ehr_create_command"
 
 source /pixl/venv/bin/activate
 
-python3 /pixl/create_pixl_tbls.py
+cd /pixl
+alembic upgrade head
```

Comment:
The otehr thing that we'd need to do for starting up the imaging API would be to have a check that 
if `SKIP_ALEMBIC=true`, then it doesn't run alembic (so that module level tests don't rely on postgres, but system tests do), and also set that ENV variable in the docker compose for the `imaging-api` tests